app/script/tracking/EventTrackingRepository.js

```diff
@@ -244,14 +242,14 @@ z.tracking.EventTrackingRepository = class EventTrackingRepository {
     });
   }
 
-  _is_domain_allowed_for_tracking() {
+  _isDomainAllowedForTracking() {
     if (!z.util.URLUtil.getParameter(z.auth.URLParameter.TRACKING)) {
-      for (const domain of EventTrackingRepository.CONFIG.USER_ANALYTICS.DISABLED_DOMAINS) {
+      return !EventTrackingRepository.CONFIG.USER_ANALYTICS.DISABLED_DOMAINS.some(domain => {
         if (z.util.StringUtil.includes(window.location.hostname, domain)) {
           this.logger.debug(`Tracking is disabled for domain '${window.location.hostname}'`);
-          return false;
+          return true;
```

Comment:
The `for..of` solution was better because it returned as soon as there was a disabled domain. This change lets the code always run through the whole array.

Reply:
`Array.prototype.some()` Does exactly that as well.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

Reply:
Oh. TIL :sun_with_face: